packages/components/releaseNotes/labkey/components.md

```diff
@@ -2,6 +2,10 @@
 
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version 0.6.4
+*Released*: 24 December 2019
```

Comment:
See note above about using "alpha package versions". We usually leave these version numbers and release dates as "TBD" in this file until we are done with dev and testing and actually ready to merge. Then we make one final commit and publish to update to the release version and set the date.

Reply:
👍